composer.json

```diff
@@ -64,7 +65,7 @@
         "friendsofsymfony/rest-bundle": "^2.3 || ^3.0",
         "jms/serializer-bundle": "^3.0",
         "matthiasnoback/symfony-dependency-injection-test": "^4.1",
-        "nelmio/api-doc-bundle": "^2.13.3",
+        "nelmio/api-doc-bundle": "^3.6",
```

Comment:
This does not match the conflict. You start testing with version 3.6, but allow all versions starting at 3.0